Components/Card/Card.css

```diff
@@ -0,0 +1,46 @@
+
+body{
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-family: Arial, Helvetica, sans-serif
+}
+
+#card{
+height: 450px;
+width: 230px;
+border: solid 2px black;
+border-radius: 20px;
+box-shadow: 2px 2px 8px dimgrey;
+}
+img{
```

Comment:
Better you give a class name to your elements, as to apply the CSS on the html-tag directly. Maybe there will be other images on the page.

Check out this metrology: https://getbem.com/